src/lib/rule2A.ts

```diff
@@ -0,0 +1,61 @@
+import {Context} from './context';
+
+/**
+ * Looks at a variety of characteristics (CSS, size on screen, attributes) 
+ * to determine if 'node' should be considered hidden
```

Comment:
I didn't actually find an authoritative definition for this but used JQuery's definition for visible (offsetHeight / offsetWidth) combined with some techniques for hiding elements with CSS. There may be more ways to consider than this but I think the offsetHeight/Width handle any case where the element doesn't take up screen space (including display:none, as far as I can tell)

Reply:
let's stick to a simpler definition for now then as this is the only one present in some spec:
 - hidden
 - aria-hidden=true
 - style="display: none;"
 - style="visibility: hidden;" 

also include the link above in the function's jsdoc so that we know where the implementation is derived from.

we can always make it more complex later if/when we see cases where the browser disagrees. But we'll probably also want to also file a bug against the spec.